### DISCUSSION TITLE
AZ SQL Servers - no option to set min TLS version

### DISCUSSION THREAD

**Discussion by ca55ar** [Azure/ResourceModules#1434] [Q&A] [answered]
I notice that in [https://github.com/Azure/ResourceModules/tree/main/arm/Microsoft.Sql/servers](url) we are unable to set the min tls version. Any chance we can get this please?

**Comment by AlexanderSehr** [comment]
Hey @ca55ar we sure can. Just created issue #1513 for it. For the future, feel free to open issues for any feature you see missing - and if you have the capacity contributions are always welcome too ;)

**Reply by ArielRam99** [reply]
@ca55ar minimum TLS version can now be set. The functionality was added in the following PR : https://github.com/Azure/ResourceModules/pull/1618

**Comment by eriqua** [comment, marked as answer]
Functionality added by @ArielRam99 in PR #1618
